portfolio/src/main/webapp/map.js

```diff
@@ -1,6 +1,30 @@
 /** Creates a map and adds it to the page. */
 function createMap() {
-  const map = new google.maps.Map(
-      document.getElementById('map'),
-      {center: {lat: 37.422, lng: -122.084}, zoom: 16});
+    const map = new google.maps.Map(
+        document.getElementById('map'),
+        {center: {lat: 37.422, lng: -122.084}, zoom: 16});
```

Comment:
Looks like you want to center to trexMarker.position?

Reply:
I refactor it to extract out as a const variable

Reply:
Fixed